hadoop-ozone/common/src/main/proto/OzoneManagerProtocol.proto

```diff
@@ -479,6 +479,7 @@ message InfoBucketResponse {
 
 message SetBucketPropertyRequest {
     required BucketArgs bucketArgs = 1;
+    optional BucketInfo bucketInfo = 2;
```

Comment:
BucketArgs and BucketInfo have a lot of common fields. Can we combine them into one proto?

Reply:
Addressed for now as making bucketArgs as optional, so that we will not set this during creating OMRequest in startTransaction, and we don't write duplicate data into raft log. 

Added a TODO for the suggested, as discussed offline.